apps/vscode.talon

```diff
@@ -1,4 +1,6 @@
-app: Code
+# see app/vscode_generic.talon for linux and windows action implementation 
+
+app: /Code.*/
```

Comment:
The best practice here is to add another entry, and for Windows include both the executable and friendly name if different. (On occasion, the MUICache can break, so the friendly name may not work for everyone.)

`app: code
app: Code.exe
app: whatever-app.name() for VSCode`

Note that separate app entries get ORed
FWIW, I use this command to get this info for Window
`debug app:
    print(app.name())
    print(app.executable())`